rbutils.gemspec

```diff
@@ -5,7 +5,7 @@ require 'rbutils/version'
 
 Gem::Specification.new do |spec|
   spec.name          = 'rbutils'
```

Comment:
if you want distribute publicly upfluence specific vendors, we should include the Upfluence name in the gem name